helpdesk-utility/src/main/java/org/symphonyoss/symphony/bots/utility/message/SymMessageBuilder.java

```diff
@@ -19,7 +19,7 @@
 
   private List<SymAttachmentInfo> attachments = new ArrayList<>();
 
-  private SymMessageBuilder(String message) {
+  protected SymMessageBuilder(String message) {
```

Comment:
You shouldn't update the class scope. Please use the 'message' static method to instantiate this class.

Reply:
ok